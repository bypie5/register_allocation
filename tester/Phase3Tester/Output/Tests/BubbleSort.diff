--- SelfTestCases/BubbleSort.out	2011-04-28 14:55:20.000000000 -0700
+++ ./Output/Tests/BubbleSort.out	2020-02-19 16:01:53.000000000 -0800
@@ -1,22 +1 @@
-20
-7
-12
-18
-2
-11
-6
-9
-19
-5
-99999
-2
-5
-6
-7
-9
-11
-12
-18
-19
-20
-0
+Program has no "Main" function.
