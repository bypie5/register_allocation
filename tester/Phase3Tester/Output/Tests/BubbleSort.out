--- Error (ProgramFault) ---
require function address, found integer10
  Main() line 20
  BBS.Start() line 39
  BBS.Init() line 225
